backend/dataall/modules/shares_base/services/share_logs_service.py

```diff
@@ -15,6 +16,7 @@
 
 class ShareLogsService:
     @staticmethod
+    @is_stack_logs_visible(targetType='shares')
```

Comment:
With the utility proposed in the comment in stack_logs_util we could rewrite this as:
```
@is_feature_enabled('modules.shares_base.features.show_share_logs', allowed_values=['enabled', 'admin_only'])
```

Reply:
Checking if a user is part of the admin group is a pretty repeated task that could be further extracted in another utility. But that is step 2 :)

Reply:
I have updated the code as per above comment. Thanks once again for a details explanation and providing a better way to do this. Also this aligns with the structure used in data.all ( base, core, modules )